sample/F8/speakers

```diff
@@ -0,0 +1,1613 @@
+[
+   {
+      "id":2,
+      "name":"Michael Slater",
+      "email":"",
+      "mobile":"",
+      "photo":"",
+      "organisation":"Facebook",
+      "sessions":[
+         {
+            "id":5,
+            "title":"Introduction to the Camera Effects Platform"
+         }
+      ],
+      "country":"",
+      "short_biography":"",
+      "website":"",
+      "twitter":"",
+      "facebook":"",
+      "github":"",
+      "linkedin":"",
+      "position":"Product Manager",
+      "city":"",
+      "gender":"male",
+      "heard_from":"",
+      "icon":"",
+      "small":"",
+      "speaking_experience":"",
+      "sponsorship_required":"",
+      "thumbnail":""
+   },
+   {
+      "id":4,
+      "name":"Lucas Rocha ",
+      "email":"",
+      "mobile":"",
+      "photo":"",
+      "organisation":"Facebook",
+      "sessions":[
+         {
+            "id":7,
+            "title":"Litho: A Declarative Framework for Efficient UIs"
+         }
+      ],
+      "country":"",
```

Comment:
no information for any speaker?
please provide the twitter or fb handles which i did on google search. see my samples